addons/point_of_sale/models/pos_session.py

```diff
@@ -1619,7 +1619,7 @@ def _get_attributes_by_ptal_id(self):
         for key, group in groupby(sorted(product_template_attribute_values, key=key), key=key):
             attribute_line_id, attribute_id = key
             values = [{**ptav.product_attribute_value_id.read(['name', 'is_custom', 'html_color'])[0],
-                       'price_extra': ptav.price_extra} for ptav in list(group)]
```

Comment:
Isn't it possible to filter all the inactive values from her directly ? This way you don't need to filter them in all the other places

Reply:
Indeed this is much more simple.